<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgRotateScreen</class>
 <widget class="QWidget" name="wdgRotateScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>128</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgRotate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_title">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Degrees</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_desc">
     <property name="toolTip">
      <string>Current degree of rotation</string>
     </property>
     <property name="text">
      <string notr="true">360Â°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="themePushButton" name="pushButton_flip">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icon/icons/switch_sides.svgz</normaloff>:/icon/icons/switch_sides.svgz</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget_directions" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_directions">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="themePushButton" name="pushButton_left">
        <property name="toolTip">
         <string>Rotate the screen 90 degrees to the left</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/icons/rotate_left.svgz</normaloff>:/icon/icons/rotate_left.svgz</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="themePushButton" name="pushButton_right">
        <property name="toolTip">
         <string>Rotate the screen 90 degrees to the Right</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/icons/rotate_right.svgz</normaloff>:/icon/icons/rotate_right.svgz</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>themePushButton</class>
   <extends>QPushButton</extends>
   <header>theme.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_left</tabstop>
  <tabstop>pushButton_right</tabstop>
  <tabstop>pushButton_flip</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
